lib/cli/cliPlugin.js

```diff
@@ -244,10 +244,10 @@ module.exports = class CLIPlugin extends CLICommand {
 
                     // Read the state
                     try {
-                        const enabled = await states.getState(pluginEnabledId)
-                            .val;
-                        if (typeof enabled === 'boolean') {
-                            CLI.success.pluginStatus(pluginName, enabled);
+                        const { val } = await states.getStateAsync(pluginEnabledId);
+
+                        if (typeof val === 'boolean') {
+                            CLI.success.pluginStatus(pluginName, hostname, instance, val);
```

Comment:
Of course, now this code is being executed and needs all for arguments, else it will print weird stuff